pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -257,6 +258,18 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
+	// Only allows live migration on SD managed clusters
```

Comment:
nit:
```suggestion
	// Only allows live migration on managed clusters
```

Reply:
Ignoring this unless theres necessary changes and i must create my own PR. I will add it in then.

Reply:
sgtm.